universum/initializer.py

```diff
@@ -0,0 +1,51 @@
+from pathlib import Path
+
+from .modules.output.output import MinimalOut
+from .lib.gravity import Module
+from .lib.module_arguments import ModuleArgumentParser
+
+__all__ = ["Initializer"]
+
+
+class Initializer(Module):
+    description: str = "Create a dummy project"
+
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
+        self.out = MinimalOut()
+
+    def execute(self) -> None:
+        target_dir = "universum_dummy_project"
+        target_dir_path = Path(target_dir)
+        script_name = "dummy_script.sh"
+        config_name = "universum_config.py"
+        artifact_name = "artifact_example.txt"
+        self.out.log("Creating a dummy project in " + target_dir)
+        target_dir_path.mkdir(parents=True, exist_ok=True)
+
+        script = target_dir_path / script_name
+        script.write_text("""#!/usr/bin/env bash
```

Comment:
Do we need to create script file at all?

Reply:
As this 'init' function is to show the user how to possibly run a project with Universum, yes, I think an example of "where to put project sources and how to handle them" will be more useful, than simply creating a sole configuration file. We have lots of those in documentation.